<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Bongos</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/bongo-drum-500x500.jpg</normaloff>:/icons/bongo-drum-500x500.jpg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>155</width>
            <height>113</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>first Drum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="drum_1">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/first drum.jpg</normaloff>:/icons/first drum.jpg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>144</width>
            <height>93</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Second Drum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="drum_2">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/first drum.jpg</normaloff>:/icons/first drum.jpg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>154</width>
            <height>100</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,10">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,15">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Virtual guitar Tune</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/guitar.png</normaloff>:/icons/guitar.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>208</width>
            <height>115</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5" stretch="10,10">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>                              Tones</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="tone_1">
              <property name="font">
               <font>
                <pointsize>19</pointsize>
               </font>
              </property>
              <property name="text">
               <string>A</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="tone_2">
              <property name="font">
               <font>
                <pointsize>19</pointsize>
               </font>
              </property>
              <property name="text">
               <string>B</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="tone_3">
              <property name="font">
               <font>
                <pointsize>19</pointsize>
               </font>
              </property>
              <property name="text">
               <string>C</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="tone_4">
              <property name="font">
               <font>
                <pointsize>19</pointsize>
               </font>
              </property>
              <property name="text">
               <string>D</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="tone_5">
              <property name="font">
               <font>
                <pointsize>19</pointsize>
               </font>
              </property>
              <property name="text">
               <string>E</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="tone_6">
              <property name="font">
               <font>
                <pointsize>19</pointsize>
               </font>
              </property>
              <property name="text">
               <string>F</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
